R/internals.R

```diff
@@ -14,7 +14,12 @@
 
   req <- httr::GET(url = api_url,          # the endpoint
                    query = parameters)
-
+  if(req$status_code == 400){
```

Comment:
Can you change this message slightly to something like: 
"HTTP 400 error encountered. Note for some endpoints the Meetup API requires you to be the respective group’s administrator."